pkg/urbit/vere/io/term.c

```diff
@@ -1036,6 +1042,169 @@ u3_term_ef_ctlc(void)
   _term_it_refresh_line(uty_u);
 }
 
+/* _term_it_put_color(): put ansi color id on lin_w. RETAINS col.
+*/
+static void
+_term_it_put_color(c3_w* lin_w,
+                   u3_noun col)
+{
+  switch ( col ) {
+    default:
```

Comment:
I think you can just remove the default case here.

Reply:
From a hoon types perspective, sure. But C can't enforce those, and I thought it was good practice to always have a default in `switch`es. It's more robust against future `+tint` changes, too; implementation assumes this function's gonna put a character there, leaving that byte of memory un-set seems not good.

Reply:
You're right, fall-thru is good behavior here. Not sure what I was thinking ...